content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/Getting_Help.adoc

```diff
@@ -2,13 +2,13 @@
 
 Should you require assistance when using this guide to deploy YubiHSM 2 on Windows, start by referencing the product documentation and currently known issues:
 
-https://developers.yubico.com/[Yubico Developers website]
+link:../../../[Yubico Developers website]
 
 https://support.yubico.com/support/home[Yubico Support]
 
-https://developers.yubico.com/YubiHSM2/[YubiHSM 2 Product Documentation]
+link:../[YubiHSM 2 Product Documentation]
```

Comment:
This should maybe be ../../ to link to the /YubiHSM2 page ?

Reply:
Yes, and the name of the ling should "YubiHSM 2 page". I put it into 
https://github.com/Yubico/developers.yubico.com/pull/249/commits/1c087fffdc46a0269c6b2c57e2121b3adb4a9ce6